JOSE_GALINDEZ/Practica2-PrincipiosSolid/README.md

```diff
@@ -0,0 +1,20 @@
+JOSE RICARDO GALINDEZ LOPEZ
+
+Practica 2 - Principios Solid 17/Nov/2021
+
+Para ejecutar el programa principal y correr las pruebas unitarias mediante la CL se
+ejecutan los siguientes comandos:
+
+Posicionarse en la ubicacion del proyecto: 
+
+cd ...\labisii2021.2\JOSE_GALINDEZ\Practica2-PrincipiosSolid\PrincipiosSOLID-SolucionTarea
+
+Para compilar el proyecto y generar el .jar
+mvn compile
+
+Para correr el proyecto
+java -cp target/PrincipiosSOLID-SolucionTarea-1.0-SNAPSHOT.jar co.unicauca.parkinglot.presentation.ClientMain
```

Comment:
No funciona:

Nov 18, 2021 10:35:04 AM co.unicauca.parkinglot.access.VehicleRepository connect
SEVERE: null
java.sql.SQLException: No suitable driver found for jdbc:sqlite::memory:
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at co.unicauca.parkinglot.access.VehicleRepository.connect(VehicleRepository.java:123)
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:105)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:36)
	at co.unicauca.parkinglot.access.RepositoryFactory.getRepository(RepositoryFactory.java:45)
	at co.unicauca.parkinglot.presentation.ClientMain.main(ClientMain.java:31)

Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.sql.Connection.createStatement()" because "this.conn" is null
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:106)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:36)
	at co.unicauca.parkinglot.access.RepositoryFactory.getRepository(RepositoryFactory.java:45)
	at co.unicauca.parkinglot.presentation.ClientMain.main(ClientMain.java:31)